src/sensors.c

```diff
@@ -64,9 +64,7 @@ static char *sensor_type_name_map[] =
 	"temperature",
 # define SENSOR_TYPE_POWER       3
 	"power",
-# define SENSOR_TYPE_CURRENT     4
-	"current",
-# define SENSOR_TYPE_UNKNOWN     5
```

Comment:
You can see this issue here: https://ci.collectd.org/job/pull-requests-build-on-epel5-x86_64-with-default-toolchain/812/console, it indeed breaks on RHEL5.